cogs/ReactForRole.py

```diff
@@ -250,6 +250,12 @@ async def rfr_edit_thumbnail(self, ctx: commands.Context):
                 await msg.edit(embed=embed)
                 await ctx.send("Okay, set the thumbnail of the thumbnail to your desired image. This will error if you "
                                "delete the message you sent with the image, so make sure you don't.")
+        elif isinstance(image, str) and (image.startswith("https://cdn") or image.startswith("https://images-ext-1.discordapp.net/external/") or image.startswith("https://media.discordapp.net/attachments/")):
```

Comment:
Would it not be better to check that it ends with .png, .jpg etc. or look for a library that checks it is a valid image.
We basically want the bot to be able to use something like `https://koalabot.uk/assets/img/KoalaBotLogo.png`

Reply:
Could check the header of a http request, for content metadata, but I didn't know how performant that would be. I'll get a change to that effect rolled out later tonight though.

Reply:
Should be resolved with 827ac9b